app/assets/javascripts/router.js

```diff
@@ -345,7 +346,18 @@ class ReactRouter extends React.Component<Props> {
               <Route path="/login" render={() => <Redirect to="/auth/login" />} />
               <Route path="/register" render={() => <Redirect to="/auth/register" />} />
               <Route path="/auth/login" render={() => <LoginView layout="horizontal" />} />
-              <Route path="/auth/register" component={RegistrationView} />
+              <Route
+                path="/auth/register"
+                render={({ location }: ContextRouter) => {
+                  const params = Utils.getUrlParamsObjectFromString(location.search);
```

Comment:
Why did you split up this method in the utils module, but then you're calling it with `location.search` anyways? Couldn't you use the original method then? Maybe I'm missing something here!

Reply:
`location` is here passed from the router (see line 351). `window.location` might have worked here, too, but to be precise, I used the value passed by react-router :)